wls-gui-wahllokalsystem/src/types/wls-types/WLSException.ts

```diff
@@ -0,0 +1,18 @@
+export default class WLSException {
+  category: string;
+  code: string;
+  message: string;
+  service: string;
+
+  constructor(
+    category: string,
+    code: string,
+    message: string,
+    service: string
+  ) {
+    this.category = category;
+    this.code = code;
+    this.message = message;
+    this.service = service;
+  }
+}
```

Comment:
```suggestion
export default class WLSException {
  constructor(
    public readonly category: string,
    public readonly code: string,
    public readonly message: string,
    public readonly service: string
  ) {}
}

```

Reply:
siehe [c288128](https://github.com/it-at-m/Wahllokalsystem/pull/633/commits/c288128c66df0cb8a67fe13d3aa31efc60ac7b0a)